diff --git a/node_modules/@react-native/community-cli-plugin/dist/commands/start/attachKeyHandlers.js b/node_modules/@react-native/community-cli-plugin/dist/commands/start/attachKeyHandlers.js
index e99086b..177c59e 100644
--- a/node_modules/@react-native/community-cli-plugin/dist/commands/start/attachKeyHandlers.js
+++ b/node_modules/@react-native/community-cli-plugin/dist/commands/start/attachKeyHandlers.js
@@ -88,9 +88,9 @@ function attachKeyHandlers({ devServerUrl, messageSocket, reporter }) {
     level: "info",
     data: `Key commands available:
 
-  ${(0, _util.styleText)(["bold", "inverse"], " r ")} - reload app(s)
-  ${(0, _util.styleText)(["bold", "inverse"], " d ")} - open Dev Menu
-  ${(0, _util.styleText)(["bold", "inverse"], " j ")} - open DevTools
+  ${(0, _util.styleText)("bold", (0, _util.styleText)("inverse", " r "))} - reload app(s)
+  ${(0, _util.styleText)("bold", (0, _util.styleText)("inverse", " d "))} - open Dev Menu
+  ${(0, _util.styleText)("bold", (0, _util.styleText)("inverse", " j "))} - open DevTools
 `,
   });
 }
